packages/env/CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 ## Master
 
+### Breaking Changes
+
+- `wp-env start` no longer accepts a WordPress branch or tag reference as its argument. Instead, create a `.wp-env.json` file and specify a `"core"` field.
```

Comment:
If I'm reading the code right, I think the previous behavior was to always detect and link the current directory whether or not it contained a config file, but the new behavior is to link the current directory only if there is no config file. Maybe this should be added as a breaking change since you have to manually specify `"."` now to include the current directory?

Reply:
No, because we didn't have a config file at all before.

Reply:
If you run `wp-env start` in a directory that does not contain `.wp-env.json`, the behaviour is the same as before.  _Technically_ there's a change in behaviour for any projects that, for some reason, have an unrelated file named `.wp-env.json`—but that's enough of an edge case that I'm okay with not noting it in the changelog.

Reply:
Fair enough, I forgot the previous version was `wp-env.json`, not `.wp-env.json`. Plus, it was probably not used widely. :)